### PR TITLE
Add tooltip of modified by

### PR DESCRIPTION
**Pull request by RonakParmar**
#### Summary of Changes

Added tooltip for "Modified by" field in Banner, Contact and Newsfeed page.
#### Testing Instructions
- Login at administrator side.
- Check banner, contact and newsfeed new/edit page->Publishing tag ->Modified by field.
  There is no tooltip for this field.
  Apply this PR and follow above steps, you can see tooltip for this field.

### CODE REVIEW

**Comment by RonakParmar** [comment]
You can see tooltip for "Modified by" field of Article, Category and Tag new/edit page in this PR #11144 . @kalpeshtailored has already added tooltip for this field for these pages.

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 1]
```diff
@@ -291,6 +291,7 @@ COM_CONTACT_WARNING_PROVIDE_VALID_NAME="Please provide a valid name."
 COM_CONTACT_WARNING_PROVIDE_VALID_URL="Please provide a valid URL."
 COM_CONTACT_WARNING_SELECT_CONTACT_TOPUBLISH="Please select a contact to publish."
 COM_CONTACT_XML_DESCRIPTION="This component shows a listing of contact information."
+COM_CONTACT_FIELD_MODIFIED_BY_DESC="Name of the user who modified this contact."
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_contact.ini]: please alpha order language strings

File: administrator/language/en-GB/en-GB.com_newsfeeds.ini [hunk 2]
```diff
@@ -128,5 +128,6 @@ COM_NEWSFEEDS_TIP_ASSOCIATED_LANGUAGE="%s %s (%s)"
 COM_NEWSFEEDS_UNPUBLISH_ITEM="Unpublish News Feed"
 COM_NEWSFEEDS_WARNING_PROVIDE_VALID_NAME="Please provide a valid name."
 COM_NEWSFEEDS_XML_DESCRIPTION="This component manages RSS and Atom news feeds."
+COM_NEWSFEEDS_FIELD_MODIFIED_BY_DESC="Name of the user who modified this newsfeed."
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_newsfeeds.ini]: please alpha order language strings

**Comment by DhavalTailored** [comment]
I have tested this item :white_check_mark: successfully on 0668e83d12eddcb23b2e0af8730e96ca9f5f830f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by tomartailored** [comment]
I have tested this item :white_check_mark: successfully on 0668e83d12eddcb23b2e0af8730e96ca9f5f830f<br /><br />I have tested this item successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 1c0bc49194057a6041c918c43f30191184f6e651<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

File: administrator/language/en-GB/en-GB.com_newsfeeds.ini [hunk 3]
```diff
@@ -44,6 +44,7 @@ COM_NEWSFEEDS_FIELD_IMAGE_CAPTION_LABEL="Caption"
 COM_NEWSFEEDS_FIELD_LANGUAGE_DESC="Assign a language to this news feed."
 COM_NEWSFEEDS_FIELD_LINK_DESC="Link to the news feed. IDN (International) Links are converted to punycode when they are saved."
 COM_NEWSFEEDS_FIELD_LINK_LABEL="Link"
+COM_NEWSFEEDS_FIELD_MODIFIED_BY_DESC="Name of the user who modified this newsfeed."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_newsfeeds.ini]: Please update this to news feed - two words https://joomla.github.io/user-interface-text/?user-interface-text/words2watch.md

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 1c0bc49194057a6041c918c43f30191184f6e651<br /><br />Please make sure newsfeed is written as two words https://joomla.github.io/user-interface-text/?user-interface-text/words2watch.md<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 1c0bc49194057a6041c918c43f30191184f6e651<br /><br />Please make sure newsfeed is written as two words https://joomla.github.io/user-interface-text/?user-interface-text/words2watch.md<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by NeerajTailored** [comment]
I have tested this item :white_check_mark: successfully on fca1f9e1939b891cd6efea2fd7ddbe5016f9f117<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by JamalTailored** [comment]
I have tested this item :white_check_mark: successfully on fca1f9e1939b891cd6efea2fd7ddbe5016f9f117<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by truptikagathara** [comment]
I have tested this item :white_check_mark: successfully on fca1f9e1939b891cd6efea2fd7ddbe5016f9f117<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11145">issues.joomla.org/joomla-cms/11145</a>.</sub>

**Comment by zero-24** [comment]
@joomla-cms-bot are you sleeping now?

**Pull request merged**